src/routes/Chat/LeaveMessage/component.js

```diff
@@ -0,0 +1,195 @@
+import { h, Component } from 'preact';
+
+import { Button } from '../../components/Button';
+import { ButtonGroup } from '../../components/ButtonGroup';
+import { Form, FormField, SelectInput, TextInput, Validations } from '../../components/Form';
+import Screen from '../../components/Screen';
+import { createClassName, sortArrayByColumn } from '../../components/helpers';
+import I18n from '../../i18n';
+import styles from './styles.scss';
+
+
+const defaultTitle = I18n.t('Leave a message');
```

Comment:
I'm confused on this, if we're disabling livechat why are we titling with 'leave a message'?